PhysicsTools/PatAlgos/test/patTuple_addBTagging_cfg.py

```diff
@@ -104,6 +104,9 @@
     ,'pfCombinedMVABJetTags'
     ,'pfPositiveCombinedMVABJetTags'
     ,'pfNegativeCombinedMVABJetTags'
+    #CTagging
+    ,'pfCombinedTMVACtagLJetTags'
```

Comment:
Unit tests are failing due to unsupported tagger names. The names here should be changed to `pfCombinedCvsLJetTags` and `pfCombinedCvsBJetTags`.